cli/render-dedupe-check-issues/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@pnpm/cli.render-dedupe-check-issues",
+  "description": "Visualize pnpm dedupe --check issues.",
+  "private": true,
```

Comment:
why is it a private package? It should be public with v0.0.0 and a changeset should be added for it to bump the version to 1

Reply:
Definitely a mistake. Thanks for catching! Should be fixed now.